pkg/k8smanifest/verify.go

```diff
@@ -0,0 +1,77 @@
+package k8smanifest
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+
+	"github.com/ghodss/yaml"
+	"github.com/sigstore/k8s-manifest-sigstore/pkg/k8smanifest"
+	k8ssigutil "github.com/sigstore/k8s-manifest-sigstore/pkg/util"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func verify(manifestPath, keyPath string) {
```

Comment:
I tested this function locally and it is working fine. But I think in kyverno we need the functionality to pass the 'keys' (file content and not the filepath)  to this function. And if this is the case then we'll have to add a lot of code in kyverno to make this function work with keyContent.

am I thinking right? or keypath will also work?

Reply:
@Namanl2001 - Kyverno will not see YAML files but will receive an admission request with the resource. The annotations will contain the message and signature. Kyverno will then need to decrypt the message and compare the YAML in there with the resource.